package.json

```diff
@@ -49,7 +49,7 @@
     "react-redux": "~4.4.5",
     "react-router": "~2.5.1",
     "react-router-redux": "~4.0.5",
-    "react-shortcuts": "~1.3.1",
+    "react-shortcuts": "git+https://github.com/Dunkat/react-shortcuts.git#dev",
```

Comment:
This is probably temporary because PR was sent to this libs` maintainer. If he won't accept or there will be compliments we will move this to mf organisation. Is it fine for you to wait for 1-2 weeks?